Assets/LeapMotion/Core/Scripts/XR/LeapXRCameraControl.cs

```diff
@@ -0,0 +1,89 @@
+/******************************************************************************
+ * Copyright (C) Leap Motion, Inc. 2011-2017.                                 *
+ * Leap Motion proprietary and  confidential.                                 *
+ *                                                                            *
+ * Use subject to the terms of the Leap Motion SDK Agreement available at     *
+ * https://developer.leapmotion.com/sdk_agreement, or another agreement       *
+ * between Leap Motion and you, your company or other organization.           *
+ ******************************************************************************/
+
+using UnityEngine;
+using UnityEngine.Rendering;
+using System;
+
+namespace Leap.Unity {
+
+  /// <summary>
+  /// Provides a handful of VR related camera utilities, such as controlling IPD and
+  /// camera distance.
+  /// </summary>
+  public class LeapXRCameraControl : MonoBehaviour {
```

Comment:
Would recommend renaming this to something like `LeapEyeDislocator` or something, since that's all it's really used for now.

Reply:
Sounds good!